src/components/CallView/Grid/Grid.vue

```diff
@@ -268,6 +270,10 @@ export default {
 			// Timer for the videos bottom bar
 			showVideoOverlayTimer: null,
 			debounceMakeGrid: () => {},
+			videosTempPromoted: [],
```

Comment:
Since it's based on speaking, shall we rename it to:
- `promotedModels`
- `tempPromotedModels`
- e.t.c?